### ISSUE TITLE
dependabot: Exclusive examples handling necessary ?

### ISSUE THREAD

**Issue by mfranzke** [db-ui/mono#621]
Why is handling the [examples folders](https://github.com/db-ui/mono/blob/7c2ed60fae22de8109a7565c134ffdfb081c702c/.github/dependabot.yml#L55-L75) differently from the other folders necessary? Or is this because dependabot wouldn't catch up non-workspace folders?
Currently this leads to redundant PRs created by dependabot.

**Comment by nmerget** [comment]
Yes they are not part of the workspace, so dependabot wouldn't update them
